src/NavBar.jsx

```diff
@@ -122,31 +122,48 @@ class NavBar extends Component {
     this.state = {
       menuClick: false,
     };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({
+      menuClick: !this.state.menuClick,
+    });
   }
 
   render() {
     return (
       <NavContainer>
         <Nav>
           <MenuContainer>
-            <DropdownBtn
-              onClick={() =>
-                this.setState({
-                  menuClick: !this.state.menuClick,
-                })
-              }
-            >
+            <DropdownBtn onClick={this.handleClick}>
               <i className="fa fa-bars" />
             </DropdownBtn>
             <Title>Water You Talking About?</Title>
           </MenuContainer>
         </Nav>
         <DropNav>
           <DropdownMenu show={this.state.menuClick} id="button">
-            <HLink to="/">Home</HLink>
-            <HLink to="/calculator">Calculator</HLink>
-            <HLink to="/prices">Prices</HLink>
-            <HLink to="/statistics">Statistics</HLink>
+            <HLink
+              onClick={() =>
```

Comment:
this should use your `handleClick` function

Reply:
Why? the function is working the way it is now
can you please explain this?